storage/rocksdb/ha_rocksdb.cc

```diff
@@ -5738,11 +5840,15 @@ int ha_rocksdb::create_key_def(const TABLE *const table_arg, const uint &i,
     kv_version = Rdb_key_def::PRIMARY_FORMAT_VERSION_UPDATE1;
   });
 
+  uint32 index_flags = (ttl_duration > 0 ? Rdb_key_def::TTL_FLAG : 0);
```

Comment:
This is where the index_flags are generated. Then they are stored. Do we still need generate_index_flags_bitmap()? Effectively the two lines of code perform the same function. Can we consolidate it down to one callsite?